NadekoBot.Core/Modules/Utility/Utility.cs

```diff
@@ -311,6 +315,62 @@ public async Task Stats()
                 await ctx.Channel.SendMessageAsync(result.TrimTo(2000)).ConfigureAwait(false);
         }
 
+        [NadekoCommand, Usage, Description, Aliases]
+        [RequireContext(ContextType.Guild)]
+        [BotPerm(GuildPerm.ManageEmojis)]
+        [UserPerm(GuildPerm.ManageEmojis)]
+        [Priority(2)]
+        public Task EmojiAdd(string name, Emote emote)
+            => EmojiAdd(name, emote.Url);
+
+        [NadekoCommand, Usage, Description, Aliases]
+        [RequireContext(ContextType.Guild)]
+        [BotPerm(GuildPerm.ManageEmojis)]
+        [UserPerm(GuildPerm.ManageEmojis)]
+        [Priority(1)]
+        public Task EmojiAdd(Emote emote)
+            => EmojiAdd(emote.Name, emote.Url);
+
+        [NadekoCommand, Usage, Description, Aliases]
+        [RequireContext(ContextType.Guild)]
+        [BotPerm(GuildPerm.ManageEmojis)]
+        [UserPerm(GuildPerm.ManageEmojis)]
```

Comment:
This means I don't need to manage permissions on this command separately, cause it will already check the user's permissions for manage emojis?

Reply:
yes, it calls this attribute https://discordnet.dev/api/Discord.Commands.RequireUserPermissionAttribute.html